packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -68,6 +68,7 @@ export interface VercelServerlessConfig {
 	analytics?: boolean;
 	imageService?: boolean;
 	imagesConfig?: VercelImageConfig;
+	useSquooshDev?: boolean;
```

Comment:
Would it make sense to change this as `devImageService: 'sharp' | 'squoosh'` instead? In case Astro supports a new image transformer in the future, we can reserve it here.

Reply:
That is definitely a better thing to do, thank you for not letting me ship this mistake